rules/linux/lnx_network_service_scanning.yml

```diff
@@ -0,0 +1,31 @@
+title: Linux Network Service Scanning
+id: 3e102cd9-a70d-4a7a-9508-403963092f31
+status: experimental
+description: Detects enumeration of local or remote network services.
+author: Alejandro Ortuno, oscd.community
+date: 2020/10/21
+references:
+  - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1046/T1046.md
+logsource:
+  category: process_creation
+  product: linux
+detection:
+  selection_1:
+    ProcessName|endswith:
+      - '/cat'
```

Comment:
most probably that one is redundant. I've checked the ART test and didn't find anything about `cat`.
could you please remove it for both rules?